lib/1.4/dml-builtins.dml

```diff
@@ -2785,6 +2785,12 @@ template field is (object, _bitsize, _lsb, init_val, shown_desc, _get, _set) {
         set_val(init_val);
     }
 
+    shared method _full_name() -> (const char *) {
```

Comment:
I'd ask for a comment on return value ownership, but I can see that we lack that on _qname as well so no need to fix now.